src/current/_includes/releases/v23.2/v23.2.0-beta.2.md

```diff
@@ -7,22 +7,33 @@ Release Date: December 5, 2023
 <h3 id="v23-2-0-beta-2-general-changes">General changes</h3>
 
 - CockroachDB now periodically dumps the state of its internal memory accounting system into the `heap_profiler/` directory when a heap profile is taken. To disable this behavior, set the `diagnostics.memory_monitoring_dumps.enabled` [cluster setting](https://cockroachlabs.com/docs/v23.2/cluster-settings) to `false`. [#114998][#114998]
+- Multi-level compactions have been disabled to investigate possible performance issues with foreground throughput and latency. [#115481][#115481]
 
 <h3 id="v23-2-0-beta-2-enterprise-edition-changes">Enterprise Edition changes</h3>
 
 - When using [Physical Cluster Replication](https://cockroachlabs.com/docs/v23.2/physical-cluster-replication-overview), you can now [initiate a cutover](https://cockroachlabs.com/docs/v23.2/cutover-replication) as of `LATEST` before the initial scan completes. [#115101][#115101]
+- Sensitive information such as `api_secret`, `sasl_password`, `client_cert`, and `ca_cert`, is now redacted in output from commands `SHOW CHANGEFEED JOB`, `SHOW CHANGEFEED JOBS`, and `SHOW JOBS`. [#115567][#115567]
+- The `physical_replication.frontier_lag_nanos` metric and the related DB Console graph have been removed because they sometimes display incorrect information. For alerting, it is recommended to use the new metric `physical_replication.replicated_time_seconds` metric instead. [#115234][#115234]
+- Fixes a bug in physical cluster replication where replicating from a source cluster that is on a version prior to v23.2.x to a destination cluster running on v23.2.x could fail because of an undefined builtin function in the source cluster. [#114257][#114257]
```

Comment:
```suggestion
- Fixed a bug in [physical cluster replication](https://cockroachlabs.com/docs/v23.2/physical-cluster-replication-overview) where replicating from a primary cluster that is on a version prior to v23.2.x to a standby cluster running on v23.2.x could fail because of an undefined builtin function in the primary cluster. [#114257][#114257]
```